### PR TITLE
polished today board

### PR DESCRIPTION
**Pull request by deewakar-k**

### CODE REVIEW

**Review by immadsahin — DISMISSED** [review]
remove 'no itmes today text'

**Comment by deewakar-k** [comment]
> remove 'no itmes today text'

- [x] done

File: apps/frontend/src/components/TodayItems.tsx [hunk 1]
```diff
@@ -13,46 +13,40 @@ interface TodayEventsProps {
...
+  const { session } = useAuth();
 
   useEffect(() => {
     if (items) {
       const combinedItems = [
         ...items.todayItems.map((item) => ({ ...item, isOverdue: false })),
         ...items.overdueItems.map((item) => ({ ...item, isOverdue: true })),
       ];
-
       const filteredItems = combinedItems.filter(item => {
```
Comment (madhav-relish) [inline apps/frontend/src/components/TodayItems.tsx]: Can we use useMemo here for filtering

File: apps/frontend/src/hooks/useEvents.ts [hunk 2]
```diff
@@ -19,6 +21,8 @@ export const useItems = () => {
         setItems(response.data)
       } catch (error) {
         console.error("failed to fetch items: ", error)
```
Comment (madhav-relish) [inline apps/frontend/src/hooks/useEvents.ts]: Not sure but would user feedback for API errors be good?

File: apps/frontend/src/app/(app)/today/page.tsx [hunk 3]
```diff
@@ -1,45 +1,37 @@
...
+import * as React from "react"
 import { ShowAgenda } from "@/src/components/atoms/ShowAgenda"
 import { TodayTextArea } from "@/src/components/TodayTextArea"
 import { TodayMeetings } from "@/src/components/TodayMeetings"
 import { DateCycle } from "@/src/components/atoms/Date"
 import { TodayItems } from "@/src/components/TodayItems"
 import { DatePicker } from "@/src/components/DatePicker"
+import usePersistedState from "@/src/hooks/usePersistedState"
 
 const TodayPage: React.FC = () => {
```
Comment (madhav-relish) [inline apps/frontend/src/app/(app)/today/page.tsx]: Try to keep app router clean and let it handle only routing.

**Review by madhav-relish — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/(app)/today/page.tsx (hunk 3 above)
Comment (deewakar-k) [reply apps/frontend/src/app/(app)/today/page.tsx]: > Try to keep app router clean and let it handle only routing.

you mean to say remove states from page?

File: apps/frontend/src/components/TodayItems.tsx (hunk 1 above)
Comment (deewakar-k) [reply apps/frontend/src/components/TodayItems.tsx]: ok!

File: apps/frontend/src/hooks/useEvents.ts (hunk 2 above)
Comment (deewakar-k) [reply apps/frontend/src/hooks/useEvents.ts]: added for debuggin in development will remove it

File: apps/frontend/src/app/(app)/today/page.tsx (hunk 3 above)
Comment (madhav-relish) [reply apps/frontend/src/app/(app)/today/page.tsx]: Yeah, just put the whole logic inside a component and render that here. Meaning the component will handle all the features and this app router page.tsx will only have to handle the routing.

**Comment by sajdakabir** [comment]
hey @madhav-relish , can you take a look into this

**Review by madhav-relish — APPROVED** [review]

**Comment by madhav-relish** [comment]
> hey @madhav-relish , can you take a look into this

Approved! @sajdakabir

**Pull request merged**